patroni/postgresql/bootstrap.py

```diff
@@ -48,7 +98,7 @@ def option_is_allowed(name):
                     if len(keys) != 1 or not isinstance(opt[keys[0]], str) or not option_is_allowed(keys[0]):
                         error_handler('Error when parsing {0} key-value option {1}: only one key-value is allowed'
                                       ' and value should be a string'.format(tool, opt[keys[0]]))
-                    user_options.append('--{0}={1}'.format(keys[0], opt[keys[0]]))
+                    user_options.append('--{0}={1}'.format(keys[0], unquote(opt[keys[0]])))
```

Comment:
shouldn't we do the same for the `options` passed as `dict`?

Reply:
Yes, I think so. It was left off before as we thought perhaps it was used only in custom bootstrap methods but on second look I think it definitely applies.

Pushed a commit for this.